components/server/src/workspace/workspace-starter.ts

```diff
@@ -244,9 +244,9 @@ export class WorkspaceStarter {
         if (user.featureFlags && user.featureFlags.permanentWSFeatureFlags) {
             featureFlags = featureFlags.concat(featureFlags, user.featureFlags.permanentWSFeatureFlags);
         }
-        // privileged and user_namespace are special cases which require the privileged-ws permission
+        // privileged are special cases which require the privileged-ws permission
         if (!this.authService.hasPermission(user, "privileged-ws")) {
-            featureFlags = featureFlags.filter(f => f != "privileged" && f != "user_namespace");
+            featureFlags = featureFlags.filter(f => f != "privileged");
```

Comment:
the `privileged-ws` permission maps to `privileged` FF, right?

Reply:
Yes, originally `privileged-ws` was the permission counterpart of the `privileged` feature flag.
For user-namespaces I originally had to weaken a user-namespaced workspace's security to a point where it felt appropriate to make it subject to the same permission as `privileged`. This is no longer the case, hence I'm reverting this restriction now.